src/components/Form/State/State.jsx

```diff
@@ -5,7 +5,105 @@ import Dropdown from '../Dropdown'
 export default class State extends ValidationElement {
   constructor(props) {
     super(props)
+    this.getStateAbbreviation = this.getStateAbbreviation.bind(this)
+    this.getStates = this.getStates.bind(this)
     this.handleError = this.handleError.bind(this)
+    this.handleUpdate = this.handleUpdate.bind(this)
+
+    this.states = [
```

Comment:
This could probably be a constant that lives outside this component, depending how many other places we use an array of states. We definitely define a couple of ways to interact with states and their abbreviations in `src/validators/location.js`

Reply:
Oh interesting file. Yeah it looks like there are a couple of different ways states are defined.

I would prefer not to duplicate the list of states/territories again in `src/validators/location.js`. So it looks like the options are to alter the `location.js` file in a way where it supports all the current use cases, or leave it as it.

In `States.jsx`, it does not have `U.S Armed Forces - Americas`, `U.S. Forces - Europe`, `U.S. Armed Forces - Pacific`. The ONLY situation where  the `States.jsx` is used, is in `MilitaryStates.jsx`, where it adds the 3 Armed Forces as `this.props.children`. It's a little strange..

Reply:
I'm not sure how you feel about it, but I'd also be fine just adding those military states...I get that they wan't to make it flexible by defining a way for `MilitaryState` to decorate it, but as you noted, the only state is `MilitaryState`. Since we only care about the concept of 'states'  within the US (as far as this form is concerned), it seems unnecessary to have that separation.

I'm also not trying to add a ton of work for you! Just thinking about how we can improve the app so we (hopefully) don't need to have these discussions ad nauseam.

Reply:
I can't tell if we are all on the same page after re-reading both your comments, but just chiming in to say that by policy the SF-86 has to indicate one of the two lists of US "state" abbreviations, depending on the specific form question. 

Some of the form questions permit the APO/FPO military states (AA, AP, AE) plus all the non-military "states" (the 50 states, plus US territories/commonwealths/possessions plus D.C. and Palau). Other questions are just supposed to permit the non-military states.